.eslintrc.cjs

```diff
@@ -3,7 +3,8 @@ module.exports = {
   extends: [
     "eslint:recommended",
     "plugin:@typescript-eslint/recommended",
-    'plugin:svelte/recommended',
+    "plugin:@typescript-eslint/recommended-type-checked",
```

Comment:
added current recommended eslint typechecking rules, and then disabled failing rules below for incremental cleanup